spec/system/home_page_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe "Home page", type: :system, driver: :selenium_chrome, js: true do
+  include_context 'home_page_setup'
+
+  scenario 'Accessing the country dropdown' do 
+    visit '/'
+
+    find_all('#v-select-search-country').first.click
+
+    expect(page).to have_all_of_selectors(
+      "#option-country-#{GeoEntity.first.id}", 
+      "#option-country-#{GeoEntity.second.id}",
+      "#option-country-#{GeoEntity.third.id}"
+    )
+  end
+
+  scenario 'Downloading set of global statistics as a zip' do 
+    visit '/'
+
+    download_link = find('a', text: 'Download statistics')
+
+    expect(download_link[:href]).to eq("#{page.current_url}downloads/global_statistics.zip")
+  end
+
+  # Not actually going to try to download the shapefiles, they are massive!
+  scenario 'Opening the spatial data downloads modal' do
+    visit '/'
+
+    find('#modal-trigger-spatial-downloads').click
+
+    expect(page).to have_css('.modal__dialog')
+
+    # Can be flaky because of the dependence on the very specific `for` attribute
+    page.find('label[for="download-radio-buttons-56-option-saltmarshes"] > .radio-button__radio').click
```

Comment:
Is this because the 56 presumably isn't always 56? Not sure how Vue's _uid is calculated... Can you match based on regex, so you don't care about the number?

Reply:
Correct - it's because the method that's generating the number is using _uid. I can have a look at matching with Regex though

Reply:
https://stackoverflow.com/questions/14635474/how-to-use-regex-in-a-capybara-finder this link is pretty useful re: CSS selectors